ansible-st2/roles/st2/tasks/3.user.yml

```diff
@@ -29,6 +29,6 @@
     mode: 0440
     regexp: "^{{ st2_system_user }} ALL="
     line: "{{ st2_system_user }} ALL=(ALL) NOPASSWD: SETENV: ALL"
-    state: present
+    state: "{{ st2_system_user_in_sudoers | ternary('present','absent') }}"
```

Comment:
Interesting that jinja provides this alternative to ``{{ 'present' if st2_system_user_in_sudoers else 'absent' }}``.